packages/@aws-cdk/cloudformation-include/test/test-templates/outputs.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Conditions": {
+    "AlwaysFalseCond": {
+      "Fn::Equals": [
+        {
+          "Ref": "AWS::Region"
+        },
+        "completely-made-up-region"
+      ]
+    }
+  },
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": "my-referenced-bucket-name"
+      }
+    }
+  },
+  "Outputs": {
+    "Output1": {
+      "Description": "the description of the output value",
+      "Value": "the output value",
+      "Export": "the export",
+      "Condition": "AlwaysFalseCond"
+    }
+  }
+}
```

Comment:
I don't think this test adds any value over `outputs-with-references.json`. I would remove this.